chromium_src/components/search_engines/template_url_prepopulate_data.cc

```diff
@@ -324,6 +415,26 @@ GetBravePrepopulatedEnginesForCountryID(
 
 }  // namespace
 
+// Substitute the refcode to keep a readable "brave" one in URL bar.
+// The refcode itself is actually the same for all users on a platform.
+bool ReplaceYandexReadableID(GURL* url) {
+  size_t pos = url->host().find("yandex.");
```

Comment:
i think this would match `yandex.somethingelse.com` and `thisisnotyandex.com`

Reply:
as well as `yandex.tld` where TLD is not registered by yandex

Reply:
Vanity code removed per discussion 👍